Microsoft.Toolkit.Uwp.UI.Controls/BladeView/BladeView.xaml

```diff
@@ -161,13 +160,12 @@
                                     Content="&#xE73F;"
                                     FontFamily="Segoe MDL2 Assets"
                                     FontSize="12" 
-                                    AutomationProperties.Name="EnlargeButton"
-                                    Foreground="{TemplateBinding CloseButtonForeground}" />
+                                    Foreground="{TemplateBinding CloseButtonForeground}" 
+                                    AutomationProperties.Name="{Binding BladeStatus}"/>
```

Comment:
Do we need to bind here if we're doing the setting in the transition code above?

Reply:
We changed this in the last commit, could you check it?